src/components/Tree/Node.tsx

```diff
@@ -170,9 +170,15 @@ export const Node = ({
                             </span>
                             {icon && <span className="tw-flex tw-justify-center tw-items-center tw-w-5">{icon}</span>}
                             {editable && onEditableSave ? (
-                                <EditableInput name={name} targetItemId={node.id} onEditableSave={onEditableSave}>
+                                <>
+                                    <EditableInput
+                                        options={{ additionalValues: node.id, enableDoubleClick: true }}
+                                        onAdditionalValueSave={onEditableSave}
+                                    >
+                                        <div className={'tw-flex'}>{name}</div>
```

Comment:
No need to use the `{}` if you're passing just a string for the tailwind classes.

Reply:
```suggestion
                                        <div className='tw-flex'>{name}</div>
```